react/Product.js

```diff
@@ -122,6 +122,7 @@ export const parseToJsonLD = (product, selectedItem, currency) => {
   const productLD = {
     '@context': 'https://schema.org/',
     '@type': 'Product',
+    '@id': product.link,
```

Comment:
Why not `product.productId`?

Reply:
@klzns The `@id` field expects a URI (or, technically, an "IRI"): https://www.w3.org/TR/2014/REC-json-ld-20140116/#node-identifiers

Reply:
oh nice